src/metabase/sync/analyze/fingerprint/insights.clj

```diff
@@ -32,7 +32,7 @@
       (cond
         (every? neg? [x1 x2])     (change (- x1) (- x2))
         (and (neg? x1) (pos? x2)) (- (change x1 x2))
```

Comment:
Unrelated to this change, but this doesn't match my intuition. `(change 1 -1)` is `2` but `(change -1 -2)` is `-0.5`? Those are both positive movements that start at a negative number. Shouldn't the change have the same sign? I would think `(change 1 -1)` should be `-2`.

Reply:
The arg order is now, before, from -1 to 1 is 200% jump, hence 2, and from -2 to -1 is an increase of 100% from the perspective of "now", hence 1. I'm not 100% if the latter is the most intuitive though; and different people measure change differently.